.github/workflows/build-native-scanner.yml

```diff
@@ -0,0 +1,273 @@
+name: Build Native (scanner)
+# TODO: DRY it up and merge with build-native.yml
+
+on:
+  pull_request:
+  push:
+    tags:
+      - '@appland/scanner-v*'
+
+jobs:
+  linux:
+    runs-on: ubuntu-latest
+
+    if:
+      ${{ contains(github.event.pull_request.labels.*.name, 'build native') ||
+      startsWith(github.ref, 'refs/tags/@appland/scanner-v') }}
+
+    env:
+      PUPPETEER_SKIP_DOWNLOAD: 1
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: ./.github/actions/setup-node
+
+      - name: Build
+        run: |
+          yarn build
+          cd packages/scanner
+          yarn build-native linux x64
+          ${GITHUB_WORKSPACE}/bin/hash release/scanner-linux-x64
+
+      - name: Publish scanner-linux-x64 artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: scanner-linux-x64
+          path: packages/cli/release/scanner-linux-x64
+
+      - name: 'Release: scanner-linux-x64'
+        uses: svenstaro/upload-release-action@v2
+        if: github.ref_type == 'tag'
+        with:
+          repo_token: ${{ secrets.GH_TOKEN }}
+          file: packages/cli/release/scanner-linux-x64
+          asset_name: scanner-linux-x64
+          tag: ${{ github.ref }}
+          overwrite: true
+
+      - name: 'Release: scanner-linux-x64.sha256'
+        uses: svenstaro/upload-release-action@v2
+        if: github.ref_type == 'tag'
+        with:
+          repo_token: ${{ secrets.GH_TOKEN }}
+          file: packages/cli/release/scanner-linux-x64.sha256
+          asset_name: scanner-linux-x64.sha256
+          tag: ${{ github.ref }}
+          overwrite: true
+
+  macos-x64:
+    runs-on: macos-12
+
+    if:
+      ${{ contains(github.event.pull_request.labels.*.name, 'build native') ||
+      startsWith(github.ref, 'refs/tags/@appland/scanner-v') }}
+
+    env:
+      PUPPETEER_SKIP_DOWNLOAD: 1
+      APPLE_IDENTITY_PRIVATE_KEY: ${{ secrets.APPLE_IDENTITY_PRIVATE_KEY }}
+      APPLE_IDENTITY_CERTIFICATE: ${{ secrets.APPLE_IDENTITY_CERTIFICATE }}
+      APPLE_CONNECT_KEY_B64: ${{ secrets.APPLE_CONNECT_KEY_B64 }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install deps
+        run: |
+          python3 -m pip install --break-system-packages setuptools
+
+      - name: Setup Node
+        uses: ./.github/actions/setup-node
+
+      - name: Build
+        run: |
+          yarn build
+          cd packages/cli
+          yarn build-native macos x64
```

Comment:
I had to update the `build-native` script to take a platform/arch in this manner.

Reply:
what do you mean? I copied this from build-native.yml

Reply:
ah, ok, I get it